### DISCUSSION TITLE
Option to Hide Exercise Illustrations and Strength/Hypertrophy Set Breakdowns (reduce visual clutter)

### DISCUSSION THREAD

**Discussion by TimKamppinen** [astashov/liftosaur#237] [Ideas]
Personally I have no use for the illustrations and they're just unnecessary visual clutter especially on the workout and history screens where they're almost too small to see anyway. Would be great to have an option to turn them off for a cleaner interface.

Similarly the Strength/Hypertrophy breakdowns like (4s,6h) etc on the week insights and the up/down arrows, to me, just make it more difficult to quickly locate the information I'm actually looking for which is the total set count for each muscle group. 

Also the strength/hypertrophy distinction is of dubious value especially with the cutoff being at 8 reps. General consensus among evidence based coaches and researchers is that anything from 5 to 30 reps is going to give a similar stimulus for hypertrophy (and even 30+ if you're actually going to true failure and not just stopping because the burn is too painful). 

But regardless, I'm sure some people find it useful, maybe if you're doing a powerbuilding program with some sets in the 1-4 range, it would just be good to have the option to hide it.

**Comment by aamaratunga** [comment]
I agree with hiding the strength/hypertrophy set breakdowns.

@astashov I'm curious what your original thinking was for adding this breakdown (like how do users use it?). And where did the cutoff of 8 reps come from?

**Reply by astashov** [reply]
I think either from Greg Nuckol's book or from Eric Helms book, don't remember...

**Reply by 0xblur** [reply]
If your focus is stength aprox 70% of your weekly sets should be in the strength rep range (1-6) and 30% in the hypertrophy range (7-25+). The ratio inverts if your focus is hypertrophy. Is not an exact science, but the ratio helps you understand what your program helps you develop more.

**Reply by John-Gee** [reply]
> I think either from Greg Nuckol's book or from Eric Helms book, don't remember...

If from Helms book, shouldn't you also have endurance at 13+?
It'd be nice to be able to define the ranges ourselves as we can with the % and make them optional as the thread asks.
